src/Bridge/Elasticsearch/DataProvider/Filter/AbstractFilter.php

```diff
@@ -142,4 +143,20 @@ protected function getMetadata(string $resourceClass, string $property): array
 
         return [$type, $hasAssociation, $currentResourceClass, $currentProperty];
     }
+
+    /**
+     * Convert CamelCase properties to snake_case.
+     *
+     * @example 'some_parent.some_child' === snakeCasePropertyPath('someParent.someChild');
+     */
+    public function snakeCasePropertyPath(string $property): string
+    {
+        $unnested = explode('.', $property);
+        $normalizer = new CamelCaseToSnakeCaseNameConverter($unnested);
```

Comment:
I think there's something wrong here as we still need to use `$this->nameConverter` here.